### ISSUE TITLE
how to use interface.h

### ISSUE THREAD

**Issue by cqkkqc** [cdcseacave/openMVS#814]
I want to texture a 3d model in  in ply format. I've check the related questions before.But I don't know how to use Interface.h to achieve this goal . Do I need to include"Interface.h" ? Or I just refer to the format and include the relevant header files （like Scene.h,）and their functions I used?

**Comment by cdcseacave** [comment]
check how the interface is used in OpenMVG: https://github.com/openMVG/openMVG/blob/develop/src/software/SfM/export/main_openMVG2openMVS.cpp

**Comment by cqkkqc** [comment]
> check how the interface is used in OpenMVG: https://github.com/openMVG/openMVG/blob/develop/src/software/SfM/export/main_openMVG2openMVS.cpp

Thanks for your kindly help. But I still have a question . How do I use Load(xxx.ply) as you said here? (https://github.com/cdcseacave/openMVS/issues/225#issuecomment-310043934) . I try to copy it in my Interface.h. But  it seems that I still need to copy a lot of other structs(like  PLY..)

**Comment by cdcseacave** [comment]
a PLY is not enough, you need camera poses, etc

**Comment by cqkkqc** [comment]
> a PLY is not enough, you need camera poses, etc
Yes ，I have prepared camera  poses and pictures,andThanks to your help,I have known how to import them into scene.mvs .But I don't know how to import my model  in ply format .

**Comment by cdcseacave** [comment]
I do not understand what you want ot do, pls elaborate

**Comment by cqkkqc** [comment]
I want to generate a scene in mvs format to texture my model (.ply) with the TextureMesh function. The data I have consists of 1, a textureless model 2, a bunch of pictures of this object taken at different angles and the corresponding pose parameters

**Comment by cdcseacave** [comment]
so once you generate the `scene.mvs` file with all the pictures and their positions, nothing else, call:
```
TextureMesh scene.mvs --mesh-file <path-to-ply>
```

**Comment by cdcseacave** [comment]
See \libs/MVS/Interface.h\ for the C++ interface API documentation and \scripts/python/MvsUtils.py\ for Python usage examples.
